docs/_data/videos.yaml

```diff
@@ -3516,11 +3514,18 @@ ro-crates/workflowhub/tutorial:
   tags: [ro-crate, workflowhub]
   title: Registering Galaxy workflows in WorkflowHub
   type: Tutorial
-  versions: []
+  versions:
+  - captions: []
```

Comment:
captions are mandatory for smorgasbord, it looks like someone's done them (fantastic!!), do you perhaps know whom? We should record that here. (otherwise this video will be marked as 'inaccessible' which isn't accurate but we just don't want to deal with checking properly via the youtube api)

Reply:
I've added support for listing institutions as caption contributors, in case it is unknown which individual provided the captions. I've set the value for this one to Australian biocommons, feel free to change if needed

Reply:
I think those captions were auto-genereated by Youtube so they may not be ideal; listing biocommons is OK for now as I don't know who uploaded.